applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_filter_element/test_thermal_filter_element_2D2N/ProjectParameters.json

```diff
@@ -52,14 +51,12 @@
     "desired_iterations"                 : 2,
     "max_radius_factor"                  : 10.0,
     "min_radius_factor"                  : 0.1,
-    "calculate_reactions"                : true,
```

Comment:
Is this intentially deleted? In thermal analysis, I expect the reaction computation to compute nodal reaction flows at the Dirichlet boundary conditions and zero nodal flow and other nodes that are without Dirichlet and Neumann boundary conditions.

Furhter, I think remove the "calculate_reactions" : true,  has no effect. The defaults in geomechanics_U_Pw_solver.py and geomechanics_solver.py, geomechanics_Pw_solver.py and geomechanics_T_solver.py are all true.